data/hai/hai outfits.txt

```diff
@@ -399,6 +399,27 @@ outfit "Chameleon Anti-Missile"
 
 
 
+outfit "Hai Cuttlefish Jammer"
+	category "Systems"
+	cost 260000
+	thumbnail "outfit/hai jammer"
+	"mass" 10
+	"outfit space" -7
+	"optical jamming" 15
+	"energy consumption" .8
+	description "When the Hai first started fighting each others there quickly was a need for a counter to trackers. The Cuttlefish Jammer will notably disturb the tracker's lock to the ship, by presenting it with a seemingly better target. This jammer is mostly needed for larger ships, that often cannot field enough anti-missiles to effectively counter trackers."
```

Comment:
```suggestion
	description "When the Hai first started fighting each other, the need for a counter to trackers quickly became apparent. The Cuttlefish Jammer can noticebly disturb a tracker's lock to the ship by presenting it with a seemingly better, albeit illusory, target. This jammer is most effective on larger ships that cannot field enough anti-missiles to effectively counter them from all directions."
```
I feel like we could have a better opener here if it were more specific. @MasterOfGrey do we have the initiation of ongoing conflict with the Unfettered mapped out? At least in terms of when/who fighting started?